src/hooks/useSpaceStyle.ts

```diff
@@ -84,7 +84,7 @@ export const useSpaceStyle = (props: SpaceProps): ViewStyle => {
   const horizontalAlignment = leftFill && rightFill ? 'center' : rightFill ? 'flex-start' : leftFill ? 'flex-end' : undefined
   const verticalAlignment = topFill && bottomFill ? 'center' : bottomFill ? 'flex-start' : topFill ? 'flex-end' : undefined
   const alignItems = sideways ? verticalAlignment : horizontalAlignment
-  const justifyContent = sideways ? horizontalAlignment : verticalAlignment
+  const justifyContent = sideways ? horizontalAlignment ?? 'space-between' : verticalAlignment
```

Comment:
Where is this used and what UI is affected? Doesn't seem like any Card work uses it

Reply:
This was suppose to include `fill` (or the expand property) as a prediciate. I made a mistake, thanks for mentioning the concern to bring this bug into light. 

I auditted that there are only two places in the LoanDetailsScene where `sideways` is used and this shouldn't affect these instances considering they aren't using `fill`.

Reply:
How do I add these loan scenes to our snapshot testing so this would have threw a red flag?